Add doc about the new tabs features (#573)
PrestaShop/docs: src/content/1.7/modules/core-updates/1.7.7.md

You can read this FAQ if you want to know [how to override the Product cover]({{< ref "1.7/faq/product#product-cover" >}}).
{{% /notice %}}

## Modern Tabs for modern modules

Modern modules that use Symfony controller and routing can now use the `$tabs` property in their module class to [automatically register Tabs in the BackOffice menu]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-tab-registration" >}}).

> Another new feature for Tabs, when a module is enabled/disabled all its related [Tab links are automatically show/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).

Reviewer: ```suggestion
Also new in this version, when a module is enabled/disabled all its related [Tab links are automatically shown/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).
```

Author: Done
